src/helpers/get-products-cart-amount.ts

```diff
@@ -0,0 +1,5 @@
+export const getProductsCartAmount = (): number => {
+    const cartIds: string[] = JSON.parse(localStorage.getItem('cartIds') || '[]');
```

Comment:
For better clarity, you can change the name of total amount of products' ids. Ex:` productsIds`, `itemsIds` insted of `cartIds`

The same for `getProductsCartAmount`, you may use `getProductsAmount` or `getItemsAmount` or `getProductsQuantity`